### PR TITLE
Set up docker AWS files

### PR DESCRIPTION
**Pull request by omcmenamin**

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    - name: Login to Amazon ECR
+      id: login-ecr
+      uses: aws-actions/amazon-ecr-login@v1
+
+    - name: Build & push the Docker image
+      env:
+        API_URL: ${{ secrets.API_URL }}
+        ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+        ECR_REPOSITORY: employee_demo
+        IMAGE_TAG: team2-web
```
Comment (shaunganley) [inline .github/workflows/docker.yml]: change to `belfast-team-2-web`

File: .github/test
Comment (shaunganley) [inline .github/test:1]: can be deleted

**Review by shaunganley — CHANGES_REQUESTED** [review]

File: .github/workflows/docker.yml (hunk 1 above)
Comment (omcmenamin) [reply .github/workflows/docker.yml]: fixed

File: .github/test
Comment (omcmenamin) [reply .github/test:1]: deleted

**Review by shaunganley — APPROVED** [review]

**Review by SeanMcAvoy29 — APPROVED** [review]

**Pull request merged**